docs/sriov-secondary-interface.md

```diff
@@ -0,0 +1,54 @@
+# SR-IOV Secondary Interface Support
+
+This adds SR-IOV support for secondary interfaces in Antrea. API support is
```

Comment:
This sounds like a commit message :)
Maybe you could rephrase it like: This doc is about SR-IOV support for secondary interfaces in Antrea...

Reply:
Will update and merge doc later. Thanks.